src/NuGet.Packaging/PackagesConfigReader.cs

```diff
@@ -122,11 +123,21 @@ public NuGetVersion GetMinClientVersion()
         }
 
         /// <summary>
-        /// Reads all package node entries in the config
+        /// Reads all package node entries in the config.
+        /// If duplicate package ids exist an exception will be thrown.
         /// </summary>
-        /// <returns></returns>
         public IEnumerable<PackageReference> GetPackages()
         {
+            return GetPackages(allowDuplicatePackageIds: false);
+        }
+
+        /// <summary>
+        /// Reads all package node entries in the config.
+        /// </summary>
+        /// <param name="allowDuplicatePackageIds">If True validation will be performed to ensure that 
+        /// only one entry exists for each unique package id.</param>
+        public IEnumerable<PackageReference> GetPackages(bool allowDuplicatePackageIds)
```

Comment:
@yishaigalatzer are there plans to expose this as an option to the restore command?